docs/setup/pick-right-solution.md

```diff
@@ -47,7 +47,7 @@ a Kubernetes cluster from scratch.
 
 * [AppsCode.com](https://appscode.com/products/cloud-deployment/) provides managed Kubernetes clusters for various public clouds, including AWS and Google Cloud Platform.
 
-* [KUBE2GO.io](https://kube2go.io) get started with highly available Kubernetes clusters on multiple public clouds along with useful tools for development, debugging, monitoring.
+* [KUBE2GO.io](https://kube2go.io) get started with highly available Kubernetes clusters on multiple public clouds along with useful tools for development, debugging, monitoring. (Kube2go has moved to Platform9 Sandbox.)
```

Comment:
Rather than referencing https://kube2go.io (which has an incorrect SSL cert fronting it) that says it's moving to Platform9's sandbox, perhaps we should just reference the "Platform9" link, make any relevant updates to the description, and remove this entry.